gatsby-node.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * Implement Gatsby's Node APIs in this file.
+ *
+ * See: https://www.gatsbyjs.org/docs/node-apis/
+ */
+
+// You can delete this file if you're not using it
```

Comment:
I understand these are just the defaults you left in, just wondering if we'll ever use them

Reply:
Yeah I think it's safe to delete this now, we are not going to use it any time soon here.